flow/Platform.actor.cpp

```diff
@@ -3623,6 +3622,12 @@ void platformInit() {
 #endif
 }
 
+std::vector<std::function<void()>> g_crashHandlerCallbacks;
+
+void registerCrashHandlerCallback(void (*f)()) {
```

Comment:
Nice! I like `registrationHandler` better than bunch of methods added to this routine. Thanks for making this change.